manager2/src/app/admin/project/project.component.html

```diff
@@ -34,6 +34,10 @@ <h4>Project <b>{{project.id}}</b></h4>
               <label for="update_project_size">Size (GB)</label>
               <input id="update_project_size" type="text" [ngModelOptions]="{standalone: true}" [(ngModel)]="project.size" class="form-control"/>
             </div>
+            <div class="col-sm-2">
+              <label for="project_exp">Creation date</label>
+              <input type="date" id="project_date" value="{{ date_convert(project.created_at) }}" class="form-control" readonly/>
```

Comment:
For existing project, with no creation date, what will be the result?

Reply:
![image](https://user-images.githubusercontent.com/882203/106184866-67dc0200-61a2-11eb-9474-9547c6a91e10.png)